firstvoices/backend/resources/dictionary.py

```diff
@@ -84,5 +85,32 @@ def before_import(self, dataset, **kwargs):
         if "site" not in dataset.headers:
             dataset.append_col(lambda x: str(self.site.id), header="site")
 
+    def import_row(self, row, instance_loader, **kwargs):
+        # Marking erroneous and invalid rows as skipped, then clearing the errors and validation_errors
+        # so the valid rows can be imported
+        import_result = super().import_row(row, instance_loader, **kwargs)
+        if import_result.import_type in [
+            RowResult.IMPORT_TYPE_ERROR,
+            RowResult.IMPORT_TYPE_INVALID,
+        ]:
```

Comment:
Since we always want to skip rows with any type of error, could we invert this condition to something more like `if import_result.import_type != RowResult.SUCCESS` ?

Reply:
I don't know how many types of errors or success statuses there are, I just wouldn't want to have an unexpected error type come up and miss this condition

Reply:
+1!

Reply:
Yes, so the plan I had in my mind was just to include these 2 status, `ERROR` and `INVALID` into skipped rows, and not combine all other statuses here. This leaves us room to handle other cases separately in future if required. I understand the concern, but from their `RowResult` enum, these are the only 2 statuses that are returned in case the import doesn't go according to the plan.

The other statuses are `DELETE`, `UPDATE`, `NEW`, `SKIP` for their relevant specific cases.